### DISCUSSION TITLE
11.0.0 Release Candidate 2.2

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#11994] [General]
https://www.nuget.org/packages/Avalonia/11.0.0-rc2.2

Porting guide: https://github.com/AvaloniaUI/Avalonia/wiki/Avalonia-11-Porting-Guide

**Comment by kyuranger** [comment]
What is the difference between RC1?

**Comment by cesarchefinho** [comment]
not working. 

there was an exception in setup() in appbuilder.

in a blank application created by visual studio new project wizard

**Reply by rabbitism** [reply]
what is the exception?

**Reply by MikeCodesDotNET** [reply]
Nowhere near enough information to do anything with this.

**Reply by DirkKramer** [reply]
Same over here, I get the following exception 

`System.MissingMethodException: 'Method not found: 'Avalonia.AppBuilder Avalonia.AppBuilder.UseStandardRuntimePlatformSubsystem()'.'`

at this method:

`    
    public static AppBuilder BuildAvaloniaApp()
        => AppBuilder.Configure<App>()
            .UsePlatformDetect()
            .LogToTrace();
`

I was/am updating from 'RC 1' to 'RC 2.2'

**Reply by kekekeks** [reply]
You have mixed avalonia versions in your solution

**Reply by DirkKramer** [reply]
Yep, I forgot Avalonia.Fonts.Inter. It was still in RC1. 

for some reason it didnt show up in my Updates tab in Visual studio NuGet.

Thanks for your help!

**Reply by cesarchefinho** [reply]
the expetion is the first throw in first if of setup() internal function of appbuilder

the if compares if some itens is null

i am not in front of computer to see the expetion, 
but the aplication is a blank avalonia generated by visual studio.

**Comment by hahyes** [comment]
Is this close to full release or not yet?

**Reply by timunie** [reply]
Hopefully very close, if no huge new issues arise. Massive testing is welcome.

**Reply by hahyes** [reply]
@timunie Just 2 days and full release! Hell yeah! <3

**Comment by TheMulhima** [comment]
How do i publish for mac, the old way I was using no longer works

**Reply by timunie** [reply]
does this help? https://docs.avaloniaui.net/docs/next/deployment/macOS

**Comment by tbibre** [comment]
**Edit : fixed after restarting Visual Studio**

I'm no longer able to compile AoT (win-x64), am I the only one?

> [...]\.nuget\packages\microsoft.dotnet.ilcompiler\8.0.0-preview.5.23280.8\build\Microsoft.NETCore.Native.targets(270,5): error MSB3073: The command ""[...]\.nuget\packages\runtime.win-x64.microsoft.dotnet.ilcompiler\8.0.0-preview.5.23280.8\tools\\ilc" @"obj\Release\net8.0\win-x64\native\\[MyApp].ilc.rsp"" stopped with the code -2147483645.

PublishReadyToRun is no longer working too. (PublishTrimmed + PublishSingleFile is working). I'm using .Net 8.0 preview 5.
It's a bit tricky to go back to rc1.1 (changing package fails because others are in rc2.2), so I've not yet tested if it's the version change or a change on my side.

**Reply by timunie** [reply]
does it work using NET7?

**Reply by tbibre** [reply]
Publish works, but I'm not 100% sure it is AoT (nothing in the log window about it, and startup time seems the same as in jit).

**Reply by tbibre** [reply]
Well, after restarting Visual Studio and regenerating, it works in .Net 8 again.
The fun of running a preview on a preview in a preview I guess.
